examples/od8_const_vel_one_agent.py

```diff
@@ -71,8 +71,8 @@
                       y_range=[-150, 150],
                       follow_agent_id=agent.id)
 """
-viewer = MPViewer(params=param_server,
-                  follow_agent_id=agent.id)
+# viewer = MPViewer(params=param_server,
```

Comment:
I would not change the default viewer here.

Reply:
reverted the change here